tests/console/zypper_in.pm

```diff
@@ -32,6 +46,32 @@ sub run {
     clear_console;    # clear screen to see that second update does not do any more
     assert_script_run("rpm -e $pkgname");
     assert_script_run("! rpm -q $pkgname");
+
+    if (!is_sle('<15-SP4') && !is_leap('<15.4')) {
+        # older releases than 15 don't have the --allow-unsigned-rpm switch
+        # also they have issues with this rpms being hashed by something newer than MD5
+        # and also this ZYPP_SINGLE_RPMTRANS feature flag is only available as of 15.4
+        assert_script_run 'touch /preinstall_fail';
+        my $r = script_run 'zypper -n in --allow-unsigned-rpm ' . data_url('zypper/hello0.rpm');
+        die "Unexpected zypper exit code $r - expected 8" unless (defined($r) && ($r == 8));
+        assert_script_run 'rm -f /preinstall_fail';
+
+        assert_script_run 'export ZYPP_SINGLE_RPMTRANS=1';
+
+        assert_script_run 'touch /preinstall_fail';
+        $r = script_run 'zypper -n in --allow-unsigned-rpm ' . data_url('zypper/hello0.rpm');
```

Comment:
Hey Dominik, I believe this line is failing on offline medium runs.
For instance in https://openqa.suse.de/tests/8364256#step/zypper_in/12 (and the other archs)

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14557